CHANGES

```diff
@@ -13,6 +13,15 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" https://semver.org
 
 ------------------------------------------------------------------------------
+*/*/* xiki-tempula,
+
+  * unreleased
+
+Fixes
+  - Change the dE method in u_nk2series to use the difference between two
+    lambda columns (issue #299, PR #300).
```

Comment:
Can you be more specific what *bug* was fixed — instead of the word "Change", something like "fixed <description of problem> to now use .... (note: will lead to different results than before) (issue ....)"

If I am a user and look through CHANGES I would want to understand if anything affects me.